cmd/preflight/cmd/check_container.go

```diff
@@ -435,6 +437,10 @@ func platformsToBeProcessed(cmd *cobra.Command, cfg *runtime.Config) ([]string,
 				// This must be an attestation manifest. Skip it.
 				continue
 			}
+			if !slices.Contains(allowedArchitectures(), img.Platform.Architecture) {
+				// The user has a architecture type in the manifest list that we do not support.
```

Comment:
should we log that an unsupported architecture was encountered ?

Reply:
I feel like logging here is more noise then anything, we don't log for any of the other conditions in this code. Also, since preflight only ships for the 4 arches, and OCP does as well, I think partners know we won't publish anything other then what we support.